### PR TITLE
added starter code

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

**Review by meagancheese — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.sps</groupId>
+  <artifactId>portfolio</artifactId>
```
Comment (jalexanderqed) [inline pom.xml]: This should be changed to interviewme or something similar.

**Review by jalexanderqed — CHANGES_REQUESTED** [review]
The whole target/ directory should be gitignore'd. You should probably just copy the .gitignore file from your training repository into this one. Then you'll need to remove the files from this PR (probably with `git rm`)

File: pom.xml (hunk 1 above)
Comment (apluscs) [reply pom.xml]: Done

**Review by jalexanderqed — APPROVED** [review]

**Review by acmcarther — APPROVED** [review]

**Pull request merged**